docs/root/intro/arch_overview/security/google_vrp.rst

```diff
@@ -55,9 +55,11 @@ attack surface for the initial stages of this program. We exclude any threat fro
 
 * Untrusted control planes.
 * Runtime services such as access logging, external authorization, etc.
-* Untrusted upstreams.
 * DoS attacks except as stipulated below.
-* Any filters apart from the HTTP connection manager network filter and HTTP router filter.
+* Any filters in the ``contrib`` directory.
+* Any filters that do not have ``stable`` status.
+* Any filters that do not have security_posture ``robust_to_untrusted_downstream`` or ``robust_to_untrusted_downstream_and_upstream``
+  in the most recent `manifest <https://github.com/envoyproxy/envoy/blob/HEAD/source/extensions/extensions_metadata.yaml>`_.
```

Comment:
I'd be explicit here and say that `robust_to_untrusted_downstream` filters do not qualify for untrusted upstream threat model.